src/cephadm/cephadm.py

```diff
@@ -7463,14 +7508,20 @@ def get_ceph_cluster_count(ctx: CephadmContext) -> int:
     return len([c for c in os.listdir(ctx.data_dir) if is_fsid(c)])
 
 
-def command_rm_cluster(ctx):
-    # type: (CephadmContext) -> None
+def command_rm_cluster(ctx: CephadmContext) -> None:
     if not ctx.force:
         raise Error('must pass --force to proceed: '
                     'this command may destroy precious data!')
 
     lock = FileLock(ctx, ctx.fsid)
     lock.acquire()
+    _command_rm_cluster(ctx, ctx.keep_logs, ctx.zap_osds)
+
+
+def _command_rm_cluster(ctx: CephadmContext, keep_logs: bool, zap_osds: bool) -> None:
```

Comment:
NIT, if the point of having this as a separate function from `command_rm_cluster` is that we can call it outside of the `rm-cluster` command, can we rename it to just `_rm_cluster` or something similar.